source/GameAction.h

```diff
@@ -74,6 +74,15 @@ class GameAction {
 	GameAction Instantiate(std::map<std::string, std::string> &subs, int jumps, int payload) const;
 
 
+private:
+	struct Mortgage {
+		int64_t principal = 0;
+		double interest = 0.;
```

Comment:
Could use `std::optional` here instead of `interest` + `useCreditScore`, since we now have access to it.